src/pages/LTIProviderSetup.vue

```diff
@@ -3,14 +3,15 @@
 <div class="lti-provider-setup">
     <b-alert v-if="error"
              variant="danger"
+             class="m-3"
```

Comment:
Not important at all, but this now has double margin on the sides.
```suggestion
             class="my-3"
```